src/d/actor/d_a_obj_table.cpp

```diff
@@ -4,78 +4,185 @@
-    /* Nonmatching */
+    u8 prm;
+    dBgW *coll;
+    uint coll_id;
+    u8 release;
+
+    cullMtx = mpModel->mBaseTransformMtx;
+    init_mtx();
+
+    fopAcM_setCullSizeBox(this, -80.0f, -1.0f, -80.0f, 80.0f, 205.0f, 80.0f);
+    prm = daObj::PrmAbstract(this, PRM_LINK_ID_W, PRM_LINK_ID_S);
+
+    if (((uint)prm == '\x02') && (coll = mpBgW, coll != NULL)) {
+        coll_id = (coll->GetId());
+        if (((int)coll_id >= 0) && ((int)coll_id < 0x100)) {
+            release = true;
+        } else {
+            release = false;
+        }
+        
+        if (release) {
+            dComIfG_Bgsp()->Release(coll);
+        }
+    }
+    return 1;
 }
 
 /* 00000284-000003B4       .text Mthd_Create__Q210daObjTable5Act_cFv */
 cPhs_State daObjTable::Act_c::Mthd_Create() {
-    /* Nonmatching */
+    cPhs_State phase_state;
+    u8 prm;
+    fopAcM_SetupActor(this, Act_c);
+
+    phase_state = dComIfG_resLoad(&mPhs, M_arcname);
+    if (phase_state == cPhs_COMPLEATE_e) {
+        prm = daObj::PrmAbstract((fopAc_ac_c*) this, PRM_LINK_ID_W, PRM_LINK_ID_S);
+        if (prm == 0) {
+            phase_state = MoveBGCreate(M_arcname, 9, 0, 0xffffffff);
+        } else {
+            phase_state = MoveBGCreate(M_arcname, 8, 0, 0xffffffff);
+        }
+        
+       JUT_ASSERT(0x8c, (phase_state == cPhs_COMPLEATE_e) || (phase_state == cPhs_ERROR_e));  
+    }
+    
+    return phase_state;
 }
 
 /* 000003B4-000003BC       .text Delete__Q210daObjTable5Act_cFv */
 BOOL daObjTable::Act_c::Delete() {
-    /* Nonmatching */
+    return 1;
 }
 
 /* 000003BC-00000408       .text Mthd_Delete__Q210daObjTable5Act_cFv */
 BOOL daObjTable::Act_c::Mthd_Delete() {
-    /* Nonmatching */
+    BOOL ret = MoveBGDelete();
+    dComIfG_resDelete(&mPhs, M_arcname);
+    return ret;
 }
 
 /* 00000408-00000488       .text set_mtx__Q210daObjTable5Act_cFv */
 void daObjTable::Act_c::set_mtx() {
-    /* Nonmatching */
+    PSMTXTrans(mDoMtx_stack_c::now, current.pos.x, current.pos.y, current.pos.z);
+    mDoMtx_ZXYrotM(mDoMtx_stack_c::now, shape_angle.x, shape_angle.y, shape_angle.z);
+    PSMTXCopy(mDoMtx_stack_c::now, mpModel->mBaseTransformMtx);
+    PSMTXCopy(mDoMtx_stack_c::now, M_tmp_mtx);
+    return;
 }
 
 /* 00000488-000004C4       .text init_mtx__Q210daObjTable5Act_cFv */
 void daObjTable::Act_c::init_mtx() {
-    /* Nonmatching */
+    J3DModel *model;
+    model = mpModel;
+    
+    model->mBaseScale.x = scale.x;
+    model->mBaseScale.y = scale.y;
+    model->mBaseScale.z = scale.z;
+    set_mtx();
+    return;
 }
 
 /* 000004C4-00000500       .text Execute__Q210daObjTable5Act_cFPPA3_A4_f */
-BOOL daObjTable::Act_c::Execute(Mtx**) {
-    /* Nonmatching */
+BOOL daObjTable::Act_c::Execute(Mtx** matrix) {
+    set_mtx();
+    *matrix = &M_tmp_mtx;
+    return 1;
 }
 
 /* 00000500-00000608       .text Draw__Q210daObjTable5Act_cFv */
 BOOL daObjTable::Act_c::Draw() {
-    /* Nonmatching */
+    f32 angle; // not 100% sure it's an angle, best guess
```

Comment:
Angles are pretty much always s16s, so I doubt this is right. It's better to leave the variable with a placeholder name so someone knows to fix it later rather than an incorrect name if you're not sure.